recipes/seqsero2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "SeqSero2" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "1e677166dba88224ee60f6c4d1585d7f4af4b03b72c89d800bf6fcaf595f3b29" %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/LSTUGA/seqsero2_test/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - python >=3
```

Comment:
only python and pip are needed in the host section

Reply:
Thanks, will revise it.

Reply:
@bgruening I tried to add 'salmid=0.1.23' to my dependencies, which has already been merged in bioconda recipes (https://github.com/bioconda/bioconda-recipes/pull/11868). However, the conda_build command failed to add 'salmid=0.1.23' (see the error message below). Do you know why?

03:33:19 BIOCONDA INFO (OUT) conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {'salmid=0.1.23'}
03:33:21 BIOCONDA ERROR COMMAND FAILED (exited with 1): docker run -t --net host --rm -v /tmp/tmpt8ny6mkg/build_script.bash:/opt/build_script.bash -v /home/circleci/project/miniconda/conda-bld/:/opt/host-conda-bld -v /home/circleci/project/recipes/seqsero2:/opt/recipe -e LANG=en_US.UTF-8 -e HOST_USER_ID=1001 bioconda/bioconda-utils-build-env:latest /bin/bash /opt/build_script.bash

03:33:21 BIOCONDA ERROR BUILD FAILED recipes/seqsero2
03:33:21 BIOCONDA INFO (COMMAND) conda build purge
03:33:22 BIOCONDA ERROR BUILD SUMMARY: of 1 recipes, 1 failed and 0 were skipped. Details of recipes and environments follow.
03:33:22 BIOCONDA ERROR BUILD SUMMARY: FAILED recipe recipes/seqsero2
Exited with code 1

Reply:
No one has added salmid to bioconda.

Reply:
I found a salmid recipe as below. Is possible to add it to bioconda?
http://bioconda.github.io/recipes/salmid/README.html

Reply:
That's odd, I'll try to get that built today.

Reply:
The package should be available in ~25 minutes. I'll try to hit rerun on circleci once it appears.

Reply:
It's there now, so this should pass testing.